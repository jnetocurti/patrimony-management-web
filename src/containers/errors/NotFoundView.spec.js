import React from 'react';

import { shallow } from 'enzyme';

import NotFoundView from './NotFoundView';

describe('src/containers/errors/NotFoundView', () => {
  it('should render correctly', () => {
    const component = shallow(<NotFoundView />);
    expect(component).toMatchSnapshot();
  });
});
