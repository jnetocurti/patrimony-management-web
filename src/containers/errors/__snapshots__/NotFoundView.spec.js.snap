// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/containers/errors/NotFoundView should render correctly 1`] = `
<ForwardRef
  className="makeStyles-root-1"
  title="404"
>
  <Styled(MuiBox)
    display="flex"
    flexDirection="column"
    height="100%"
    justifyContent="center"
  >
    <WithStyles(ForwardRef(Container))
      maxWidth="md"
    >
      <WithStyles(ForwardRef(Typography))
        align="center"
        color="textPrimary"
        variant="h1"
      >
        404: A página que você está procurando não existe
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        align="center"
        color="textPrimary"
        variant="subtitle2"
      >
        Ou você tentou um caminho duvidoso ou veio aqui por engano. Seja o que for, tente usar a navegação
      </WithStyles(ForwardRef(Typography))>
      <Styled(MuiBox)
        textAlign="center"
      >
        <img
          alt="Under development"
          className="makeStyles-image-2"
          src="/static/images/undraw_page_not_found_su7k.svg"
        />
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Container))>
  </Styled(MuiBox)>
</ForwardRef>
`;
